src/lib/resource/sections/applicant-fatal-claim.js

```diff
@@ -10,15 +10,17 @@ module.exports = {
             properties: {
                 'q-applicant-fatal-claim': {
                     type: 'boolean',
-                    title: 'Are you applying because someone died?',
+                    title: 'Select why you are applying',
                     oneOf: [
                         {
-                            title: 'Yes',
-                            const: true
+                            title: 'I am applying because of an injury',
+                            description:
+                                'This could include abuse over a period of time, or a disabling mental injury as a result of witnessing a crime',
+                            const: false
```

Comment:
Following on from yesterdays conversation. I think this is fine as const:false as the question is q-applicant-fatal-claim.